scripts/sentiment_analysis/index.rst

```diff
@@ -21,3 +22,181 @@ Use the following command to run with pre-trained model (`log <https://github.co
 .. code-block:: console
 
    $ python finetune_lm.py --gpu 0 --batch_size 16 --bucket_type fixed --epochs 3 --dropout 0 --lr 0.005 --valid_ratio 0.1 --save-prefix imdb_lstm_200  # Test Accuracy 86.46
+
+
+Through textCNN Model
++++++++++++++++++++++++++++++++++++++++
+
+
+This script can be used to train a sentiment analysis model with convolutional neural networks, i.e., textCNN:
+
+Kim, Y. (2014). Convolutional neural networks for sentence classification. arXiv preprint arXiv:1408.5882.
```

Comment:
Sorry but I am not suer whether you mean this paper is the source of the model in this page. If not, what about the source?
Thanks!
#1184